.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: python
+python:
+  - "2.7"
+install:
+    - pip install yapf==0.6.2
+script:
```

Comment:
I think there has to be some way of doing it, but I don't think anyone on the team knows YAML very well. Just a minor thing though, so it's not worth spending much time if the fix isn't easy.